File: 1957-delete-characters-to-make-fancy-string/1957-delete-characters-to-make-fancy-string.java
class Solution {
    public String makeFancyString(String s) {
       
      int count=1;
      StringBuilder sb=new StringBuilder();
      sb.append(s.charAt(0));
      for(int i=1;i<s.length();i++)
      {
        char ch=s.charAt(i);
        char ch2=s.charAt(i-1);
        if(ch==ch2)
        {
            count++;
        }
        else
        {
            count=1;
        }
        if(count<3)
        {
            sb.append(ch);
        }
      }
      return sb.toString();
        
    }
}

File: 1957-delete-characters-to-make-fancy-string/1957-delete-characters-to-make-fancy-string.cpp
class Solution {
public:
    string makeFancyString(string s) {
        string ans="";
        int count=1;
        int n=s.length();
         for(int i=1;i<=n;i++)
         {
            char ch=s[i-1];
            char ch2=s[i];
              if(ch==ch2)
              {
                count++;
              }
              else
              {
                count=1;
              }
              if(count<3)
              {
                ans+=s[i-1];
              }
         }
         return ans;
        
    }
};